### DISCUSSION TITLE
Open Dataset for Image reconstruction algorithm testing

### DISCUSSION THREAD

**Discussion by frmrz** [DavidT3/DAXA#156] [Q&A]
Hi all!

I'm testing a reconstruction algorithm to improve patch-based image-to-image translation using generative models. I'm searching for possible applications to have more robust testing and astronomical images are one possible target given their high resolution.

Are you familiar with open datasets in this field that might be suited for the task? Or any data scraping script for building one?

Thank you all!
Francesco

**Comment by DavidT3** [comment]
Hi Francesco, 

You will find that a lot of astronomy data is openly available, if you know where to look - though its not as monolithic as you seem to be thinking, we observe at many different wavelengths and with many different instruments, and sensitivity and resolution vary greatly between them (as well as other aspects of the images). 

I have a great interest in applying machine learning techniques to astronomy data, so I'd be interested to hear more about your work. If you want to reach out (turne540@msu.edu) we can schedule a chat and you can take me through your current project, and I can suggest some possibilities for astro data you could use.

Cheers,
David
